src/ipbb/depparser/_fileparser.py

```diff
@@ -463,13 +463,23 @@ def _parse_file(self, aPackage, aComponent, aDepFileName, aParentDep):
         self._depregistry[lDepFilePath] = lCurrentFile
 
         with open(lDepFilePath) as lDepFile:
-            for lLineNr, lLine in enumerate(lDepFile):
+            lLinePrev = ""
+            for lLineNr, lLineRaw in enumerate(lDepFile):
 
                 lDepInfo = (lCurrentFile.full_path(), lLineNr)
 
                 # --------------------------------------------------------------
                 # Pre-processing
                 try:
+                    # See if this is a split line or not.
+                    LINE_SPLIT_CHARACTER = "\\"
+                    if lLineRaw.strip().endswith(LINE_SPLIT_CHARACTER):
+                        lLinePrev += lLineRaw.strip()[:-1]
```

Comment:
Can you change the `+= ...` here to `+= " " + ...`, to be consistent with the space addition in the else clause? Without that change the following is OK:
```
src fileA.vhd \
    fileB.vhd\
    fileC.vhd
```
... whereas the following leads to an error:
```
src fileA.vhd\
    fileB.vhd \
    fileC.vhd
```

Reply:
Ok. I have made a few overall improvements to the stripping and adding of whitespace around these line continuations. This version should be more correct, and more respectful for (possibly intentional) whitespace on the left-hand side.